hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/rsql/RSQLTargetFieldValidationTest.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2020 devolo AG and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.jpa.rsql;
+
+import io.qameta.allure.Description;
+import org.eclipse.hawkbit.repository.TargetFields;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class RSQLTargetFieldValidationTest {
```

Comment:
My 5 cents to this PR: Please don't forget to add the Allure annotations `@Feature(...)` and
`@Story(....)`	to the test class :)